bin/sl-pm.js

```diff
@@ -43,6 +43,7 @@ var parser = new Parser(
     'd:(driver)',
     'l:(listen)',
     'N:(no-control)', // unused. left for backwards compat.
+    's:(skip-default-install)',
```

Comment:
`s:` makes this take an argument, but it does not. Did you test this? using `--skip-default-install` should have triggered a parse error, unless it ate whatever string followed as the "option value".